features/editor/components/editor-header.tsx

```
@@ -11,21 +11,37 @@ import {
 import { editorAtom } from "@/features/editor/store/atoms";
+import { AnimatedThemeToggler } from "@/components/ui/animated-theme-toggler";
+import { useGuestWorkflow, useUpdateGuestWorkflow } from "@/features/workflows/hooks/use-guest-workflows";
+import { Tooltip, TooltipContent, TooltipTrigger } from "@/components/ui/tooltip";
+import { Badge } from "@/components/ui/badge";
+import { useTRPC } from "@/trpc/client";
 
 export const EditorSaveButton = ({ workflowId }: { workflowId: string }) => {
   const editor = useAtomValue(editorAtom);
+  const { data: authData } = useQuery({
+    queryKey: ["session"],
+    queryFn: () => authClient.useSession(),
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)